yarn.lock

```diff
@@ -2661,6 +2649,18 @@
     tiny-warning "^1.0.3"
     toformat "^2.0.0"
 
+"@pancakeswap/uikit@^0.59.0":
```

Comment:
are you sure?  This yarn.lock file is wrong.  Why are you still using `@pancakeswap/uikit` ?